plugins/system/tour/tour.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  System.Tourguide
+ *
+ * @copyright   Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+defined('_JEXEC') or die;
+
+use Joomla\Registry\Registry;
+
+/**
+ * PlgSystemTour
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class PlgSystemTour extends JPlugin
+{
+	/**
+     * Listener for the `onBeforeRender` event
+     *
+     * @return  void
+     *
+     * @since   __DEPLOY_VERSION__
+     */
+	public function onBeforeRender()
+	{
+		// Get the application object
+		$app = JFactory::getApplication();
+	
+		// Run in backend
+		if ($app->isClient('administrator'))
+		{
+			// Get the input object
+			$input = $app->input;
+
+			// Get an instance of the Toolbar
+			$toolbar = JToolbar::getInstance('toolbar');
+		}
+	}
+
+	/**
+     * Listener for the `onBeforeCompileHead` event
+     *
+     * @return  void
+     *
+     * @since   __DEPLOY_VERSION__
+     */
+	public function onBeforeCompileHead()
+	{
+		// Only going to run these in the backend for now
+		if (JFactory::getApplication()->isClient('administrator'))
+		{
+			// Hopscotch Tour File
+			JHtml::_(
+				'script', JUri::root()
+				. 'media/plg_system_tour/js/hopscotch.min.js',
+				array('version' => 'auto', 'relative' => true)
+			);
+
+			JHtml::_(
+				'stylesheet', JUri::root()
+				. 'media/plg_system_tour/css/hopscotch.min.css',
+				array('version' => 'auto', 'relative' => true)
+			);
+			$lang = JFactory::getLanguage()->getTag();
+			$document = JFactory::getDocument();
+			$jinput = JFactory::getApplication()->input;
+			$document->addScriptDeclaration("var urlOption = '" . $jinput->get('option') . "' ;");
```

Comment:
Sorry @shivamdiehard, I did look at the right branch, just posted in the wrong place. 

This may help, https://docs.joomla.org/J3.x:Adding_JavaScript_and_CSS_to_the_page#Adding_the_options_to_your_JavaScript_code with what @dgt41 is saying.